bin/mixserv_daemon.sh

```diff
@@ -134,14 +168,14 @@ case $1 in
     if [ -f $MIXSERV_PID_FILE ]; then
```

Comment:
better to apply `yarn application -status` for YARN mode.

https://hadoop.apache.org/docs/r2.4.1/hadoop-yarn/hadoop-yarn-site/YarnCommands.html#application